express/code/blocks/faqv2/faqv2.js

```diff
@@ -0,0 +1,138 @@
+import { createTag } from '../../scripts/utils.js';
+
+function buildTableLayout(block) {
+  const parentDiv = block.closest('.section');
+  const rows = Array.from(block.children);
+  block.innerHTML = '';
+  const background = rows.shift();
+  background.classList.add('faqv2-background');
+  parentDiv.prepend(background);
```

Comment:
not sure if I'm writing a comment before you had a chance to change, so sorry if that's the case. 
you're still adding a background on the section here. Backgrounds for a section should never be set by block code, and instead by the section metadata - https://milo.adobe.com/docs/authoring/section-metadata